src/github.ts

```diff
@@ -79,30 +86,76 @@ interface GitHubPR {
 
 export class GitHub {
   octokit: Octokit;
+  request: Function;
+  graphql: Function;
   token: string | undefined;
   owner: string;
   repo: string;
   apiUrl: string;
   proxyKey?: string;
-  request: Function;
+  probotMode?: boolean;
```

Comment:
If we keep this instance variable, it looks like it's always defined.

Reply:
Because of the `?` it will be defined, but `undefined` if not provided ... didn't love this, the alternative would be perhaps dynamically defining `graphqlRequest` and `decoratePaginateOpts`, depending on whether `octokitAPis` have been provided.